packages/events/src/router/event/event.actions.declare.test.ts

```diff
@@ -121,12 +121,16 @@ test('Skips required field validation when they are conditionally hidden', async
   })
 
   const response = await client.event.actions.declare.request(data)
-  const activeActions = getAcceptedActions(response)
 
-  const savedAction = activeActions.find(
-    (action) => action.type === ActionType.DECLARE
+  const savedAction = response.actions.find(
+    ({ type, status }) =>
+      type === ActionType.DECLARE && status === ActionStatus.Requested
   )
-  expect(savedAction?.declaration).toEqual(form)
+
+  expect(savedAction?.status).toEqual(ActionStatus.Requested)
+  if (savedAction?.status === ActionStatus.Requested) {
+    expect(savedAction.declaration).toEqual(form)
+  }
```

Comment:
Won't the `if` condition always be true? Because if it was false then the test would have failed in the previous line?